kale/loaddata/dataset_access.py

```diff
@@ -28,21 +28,49 @@ def get_train(self):
         """
         raise NotImplementedError()
 
-    def get_train_val(self, val_ratio):
+    def get_class_subsampled_train(self, class_ids=None):
+        """
+        Args:
+            class_ids (list, optional): List of class ids that are to be subsampled.
```

Comment:
Here we are more choosing a subset (purposely) rather than doing subsampling. Consider to use subset, e.g. class_ids are List of the chosen subset of class ids